tests/suites/test_suite_ssl.function

```diff
@@ -908,6 +908,34 @@ int mbedtls_move_handshake_to_state( mbedtls_ssl_context *ssl,
         }                                       \
     } while( 0 )
 
+void set_ciphersuite( mbedtls_ssl_config *conf, const char *cipher,
+                      int* forced_ciphersuite )
+{
+    const mbedtls_ssl_ciphersuite_t *ciphersuite_info;
+    forced_ciphersuite[0] = mbedtls_ssl_get_ciphersuite_id( cipher );
+    forced_ciphersuite[1] = 0;
+
+    ciphersuite_info =
+        mbedtls_ssl_ciphersuite_from_id( forced_ciphersuite[0] );
```

Comment:
Will `ciphersuite_info` be NULL if the cipher ID is 0 or in some way invalid?

Reply:
Good catch, I'll add a check for that.